plugin-api/src/main/java/org/sonar/api/server/rule/RuleTagsToTypeConverter.java

```diff
@@ -30,7 +30,6 @@
 /**
  * @see org.sonar.api.server.rule.RulesDefinition.NewRule#setType(RuleType)
  * @since 5.5
- * @deprecated since 10.1.
```

Comment:
Is this change intended? I don't understand the link with the tickets.

Reply:
Indeed that is not related with the tickets and should be done separately.
I believe the marker was left by mistake when the class was un-deprecated in 10.14.

Reply:
If this is to correct a previous typo, I think you can reuse the previous ticket, no need to create a new one just for that IMO.